pyleecan/Methods/Mesh/MeshMat/get_cell_area.py

```diff
@@ -1,22 +1,39 @@
 # -*- coding: utf-8 -*-
+from numpy import abs, newaxis
 
 
 def get_cell_area(self, indices=None):
-    """Return the area of the cells on the outer surface.
-
+    """
```

Comment:
This method should be defined in the corresponding RefCell (i.e. Triangle3).
Moreover, this type of operation can be performed using jacobian product (so it could be a method of RefCell class).

Reply:
I will try to fix that.

Reply:
Don't worry about the jacobian product, I will do it latter. Just put the ShoeLace algorithm in the Triangle3 class, and call self.cell["Triangle3"].get_cell_area

Reply:
:-) Perfect. Thank you!